region/brinstar/red/Beta Power Bomb Room.json

```diff
@@ -447,6 +466,32 @@
         "In this lower position, the Sidehoppers are much less likely to die, and are more likely to hit Samus instead of the light orb."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "Gain Blue Suit (X-Mode)",
+      "requires": [
+        {"obstaclesCleared": ["A"]},
+        {"or": [
+          {"and": [
+            {"obstaclesNotCleared": ["B"]},
+            {"ammo": {"type": "PowerBomb", "count": 1}},
+            "canShinechargeMovementComplex",
+            {"canShineCharge": {"usedTiles": 29, "openEnd": 0}}
+          ]},
+          {"and": [
+            {"obstaclesCleared": ["B"]},
+            "canSamusEaterStandUp",
+            {"samusEaterFrames": 160},
+            "h_shinechargeMaxRunway",
+            "h_spikeSuitSamusEaterLeniency"
+          ]}
+        ]},
+        "h_thornXModeFramePerfectExtraLeniency",
+        "h_thornXModeBlueSuit",
+        {"shinespark": {"frames": 4}}
+      ],
+      "flashSuitChecked": true
```

Comment:
A note about getting around the yapping maw would be helpful.  Even if just for peace of mind.